base_exception/README.rst

```diff
@@ -0,0 +1,61 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
see https://github.com/OCA/server-tools/pull/1392#pullrequestreview-164502064 for remarks that can apply also here.